### PR TITLE
1178 Fix CopyItemsd for Tensor data

### LINKED ISSUE #1178: CopyItemsd transform can't support tensor from model
**Describe the bug**
Thanks for @madil90 's bug report, `CopyItemsd` transform can work on normal `torch.Tensor` data but can't work on a tensor in graph:
```
ERROR - Exception: Applying transform <monai.transforms.compose.Compose object at 0x7f0700e9deb8>.
Traceback (most recent call last):
  File "/home/madil/dlmed/code/MONAI/monai/transforms/utils.py", line 309, in apply_transform
    return transform(data)
  File "/home/madil/dlmed/code/MONAI/monai/transforms/utility/dictionary.py", line 406, in __call__
    d[new_key] = copy.deepcopy(d[key])
  File "/opt/conda/lib/python3.6/copy.py", line 161, in deepcopy
    y = copier(memo)
  File "/opt/conda/lib/python3.6/site-packages/torch/tensor.py", line 38, in __deepcopy__
    raise RuntimeError("Only Tensors created explicitly by the user "
RuntimeError: Only Tensors created explicitly by the user (graph leaves) support the deepcopy protocol at the moment
```
Need to fix it for post transforms.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #1178 .

### Description
This PR added support for `CopyItemsd` transform to copy the Tensor from graph.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by ericspod — APPROVED** [review]

File: monai/transforms/utility/dictionary.py [hunk 1]
```diff
@@ -403,7 +403,10 @@ def __call__(self, data):
         for key, new_key in zip(self.keys * self.times, self.names):
             if new_key in d:
                 raise KeyError(f"Key {new_key} already exists in data.")
-            d[new_key] = copy.deepcopy(d[key])
+            if type(d[key]) == torch.Tensor:
```
Comment (wyli) [inline monai/transforms/utility/dictionary.py:406]: this could be isinstance torch.Tensor? the type == comparison is not good for subclasses

Comment (Nic-Ma) [reply monai/transforms/utility/dictionary.py:406]: Thanks for your suggestion, let me update it.

**Pull request merged**